extensions/amp-ad-network-mytarget-impl/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/a4a'}, {name: 'mailru'}],
+    },
+  ],
+}
```

Comment:
Approving OWNERS changes. Looks like the suggested reviewer is `jridgewell`, he's set to `notify: false` so you'll have to request him as a reviewer manually once you're ready for his review (owners bot won't)

Reply:
@rcebulko, thank you for the answer.
I've added some changes to fix the code style, because TravisCI 'Checks' failed.
Does this change need additional approval?

Reply:
Is @mailru an actual user? It doesn't come up in GitHub